tabular/src/autogluon/tabular/task/tabular_prediction/predictor.py

```diff
@@ -816,6 +817,120 @@ def fit_weighted_ensemble(self, base_models: list = None, name_suffix='_custom',
+            If this has occurred, then the indices and row counts of the returned pandas Series in this method may not align with the training data.
+            In this case, consider fetching the processed training data using `predictor.load_data_internal()` instead of using the original training data.
+            A more benign version of this issue occurs when 'log_loss' wasn't specified as the eval_metric but rare classes were dropped by AutoGluon.
+            In this case, not all of the original training data rows will have an OOF prediction. It is recommended to either drop these rows during the join or to get direct predictions on the missing rows via `predictor.predict`.
+
+        Parameters
+        ----------
+        model : str (optional)
+            The name of the model to get out-of-fold predictions from. Defaults to None, which uses the highest scoring model on the validation set.
+            Valid models are listed in this `predictor` by calling `predictor.get_model_names()`
+        transformed : bool, default = False
+            Whether the output values should be of the external label representation (False) or the internal label representation (True).
+            Generally, most users will want the external representation and keep `transformed=False`.
+
+        Returns
+        -------
+        :class:`pd.Series` object of the out-of-fold training predictions of the model.
+        """
+        y_pred_proba_oof_transformed = self.get_oof_pred_proba(model=model, transformed=True)
+        y_index = y_pred_proba_oof_transformed.index
+        y_pred_oof_transformed = get_pred_from_proba(y_pred_proba=y_pred_proba_oof_transformed.to_numpy(), problem_type=self._trainer.problem_type)
+        y_pred_oof_transformed = pd.Series(data=y_pred_oof_transformed, index=y_index, name=self.label_column)
+        if transformed:
+            return y_pred_oof_transformed
+        else:
+            return self.transform_labels(labels=y_pred_oof_transformed, inverse=True, proba=False)
+
+    # TODO: Improve error messages when trying to get oof from refit_full and distilled models.
+    # TODO: v0.1 add tutorial related to this method, as it is very powerful.
+    # TODO: v0.1 This can cause very strange issues related to index mismatches with original training data on extreme edge cases (multiclass where autogluon duplicated rows in data due to rare classes and eval_metric was logloss)
+    #  This is a very rare case but needs to be fixed by v0.1 release
+    #  It might be good enough to do an inner join on training data, but it needs to be tested
+    def get_oof_pred_proba(self, model: str = None, transformed=False, as_multiclass=False) -> Union[pd.DataFrame, pd.Series]:
+        """
+        Note: This is advanced functionality not intended for normal usage.
```

Comment:
this method's documentation can probably be much shorter and just say to refer to the `get_oof_pred()` documentation

Reply:
Added